be/src/connector_sink/file_chunk_sink.h

```diff
@@ -82,8 +82,8 @@ class FileChunkSinkProvider : public ConnectorChunkSinkProvider {
 public:
     ~FileChunkSinkProvider() override = default;
 
-    StatusOr<std::unique_ptr<ConnectorChunkSink>> create_chunk_sink(std::shared_ptr<ConnectorChunkSinkContext> context,
```

Comment:
Why remove statusor? I think it is a good way to return error message when something unexpected happens

Reply:
Yes. I will put it back.